libpointmatcher_ros/CMakeLists.txt

```diff
@@ -26,10 +26,9 @@ libpointmatcher
 )
 
 # c++ 0x
-#if("${CMAKE_CXX_COMPILER_ID}" MATCHES "clang") 
-#	set(CMAKE_COMPILER_IS_CLANGXX)
-#endif ("${CMAKE_CXX_COMPILER_ID}" MATCHES "clang")
-if(CMAKE_COMPILER_IS_GNUCXX)
+if("${CMAKE_CXX_COMPILER_ID}" MATCHES "[cC]lang")
+	set(CMAKE_COMPILER_IS_CLANGXX)
```

Comment:
Yeah could just skip it -- I think before it was part of the following if/then statement but I guess it makes way more sense to just skip it entirely since it's now an if/elseif/else statement. Thanks!